packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -725,8 +725,9 @@ class ContextualMenuInternal extends React.Component<IContextualMenuInternalProp
         };
         ariaLabellledby = id;
       } else {
-        headerContextualMenuItem = sectionProps.title;
-        ariaLabellledby = this._id + sectionProps.title.text?.replace(/\s/g, '');
+        const id = sectionProps.title.text ? this._id + sectionProps.title.text.replace(/\s/g, '') : this._id;
```

Comment:
Would you mind adding a comment about why this is needed?

Reply:
added a comment